changelog/README.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 24th October 2024
+* Extended [Get all bills](../operations/bills.md#get-all-bills) response object with `Name` property.
+
+

```

Comment:
```suggestion


```
You've get rid off the first empty line but you have added new empty line at below the text, resulting in double empty line. 😁